app/scripts/app.js

```diff
@@ -55,10 +95,16 @@ app.controller('ModalInstanceCtrl', ['$scope', '$modalInstance', 'Room', functio
         rooms.$add({ displayName: roomName }).then(function(firebaseRef) {
         var id = firebaseRef.key();
         console.log("Added record with id " + id);
+        // seems like you'd ideally want a prompt here to capture the username
```

Comment:
so basically I have this modal that is able to get the username and save it which happens in the controller in line 55. 

The thing is, you want to set username as soon as a new room is created. That happens here as part of this function. I just don't know how to trigger the controller that's responsible for it though. Is that possible here?